consensus/ibftlegacy/src/main/java/tech/pegasys/pantheon/consensus/ibftlegacy/blockcreation/IbftExtraDataCalculator.java

```diff
@@ -1,42 +0,0 @@
-/*
```

Comment:
Given this is from ibftLegacy, I'd be tempted to keep this - Legacy was never completed, so will have dead code.

Reply:
What is IbftLegacy? Why keep code in the repo for functionality that was never completed?